### PR TITLE
Multiple fixes for infographics

### LINKED ISSUE #206: Infometrics CSV file is written twice
### FloodAdapt version checks

- [X] I have checked that this issue has not already been reported.
- [X] I have checked that this bug exists on the latest version of FloodAdapt.

### Reproducible Example

If you run a scenario, the metrics CSV is written, then the metrics CSV at aggregation scale, then the metrics CSV is written again.

### Current behaviour

If you run a scenario, the metrics CSV is written, then the metrics CSV at aggregation scale, then the metrics CSV is written again.

### Desired behaviour

The metrics CSV should only be generated once

### Task list

* Find out why the metrics CSV is being written a second time
* Correct the problem

### Additional context

_No response_

### LINKED ISSUE #219: move infometric and infographic methods to direct_impacts
### Kind of request

Changing existing functionality

### Enhancement Description

Right now these methos are in the scenario class but wouldn't it make more sense for them to be in the direct_impacts one, as prostprocessing steps of FIAT?

### Task list

TBD

### Use case

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by dladrichem**
Closes #219 & Closes  #206

### CODE REVIEW

File: flood_adapt/object_model/direct_impacts.py [hunk 1]
```diff
@@ -126,6 +126,8 @@ def run_models(self):
         end_time = time.time()
         print(f"Running FIAT took {str(round(end_time - start_time, 2))} seconds")
 
+        self.postprocess_fiat()
```
Comment (panosatha) [inline flood_adapt/object_model/direct_impacts.py:129]: Why was this added here? I think the whole idea was to seperate preproccess, run, and postprocess to different methods. 

The scenario.run() method takes care of the workflow.
```suggestion
```

File: flood_adapt/object_model/direct_impacts.py [hunk 2]
```diff
@@ -243,6 +245,7 @@ def run_fiat(self):
             return process.returncode
 
     def postprocess_fiat(self):
+        # Get the metrics
```
Comment (panosatha) [inline flood_adapt/object_model/direct_impacts.py:248]:
```suggestion
        # Get FIAT results
```

File: flood_adapt/object_model/direct_impacts.py [hunk 3]
```diff
@@ -373,14 +374,24 @@ def _create_infometrics(self, fiat_results_path):
...
+    def _create_infographics(self, mode):
         # Get the infographic
-        InfographicsParser().write_infographics_to_file(
-            scenario_name=self.name,
-            database_path=Path(self.database_input_path).parent,
-            keep_metrics_file=True,
+        database_path = Path(self.database_input_path).parent
+        metrics_path = database_path.joinpath(
+            "output", "infometrics", f"{self.name}_metrics.csv"
         )
```
Comment (panosatha) [inline flood_adapt/object_model/direct_impacts.py:388]: couldn't this be saved in the class when _create_infometrics is run? This way we make sure that the path is defined once.

**Comment by panosatha** [comment]
- [ ] It seems that the api that gets the location of the infographic has not been updated accordingly and the infographic is not loaded in the collapsable frame. **api.output.get_infographic()** should be changed
- [ ] I see that there is a **api.output.make_infographic()** as well. Do we need that?
- [ ] Do a merge of env_fix, because I think not everything is up to date

File: flood_adapt/object_model/direct_impacts.py (hunk 1 above)
Comment (dladrichem) [reply flood_adapt/object_model/direct_impacts.py:129]: You're right, some stuff went wrong with the rebase apparently

**Review by panosatha — CHANGES_REQUESTED** [review]
See my previous comments and in addition:

- [x] It seems that the api that gets the location of the infographic has not been updated accordingly and the infographic is not loaded in the collapsable frame. api.output.get_infographic() should be changed
- [ ]  I see that there is an api.output.make_infographic() as well. Do we need that?
- [x]  Do a merge of env_fix, because I think not everything is up to date

File: flood_adapt/object_model/direct_impacts.py [hunk 4]
```diff
@@ -373,14 +374,24 @@ def _create_infometrics(self, fiat_results_path):
         metrics_writer.parse_metrics_to_file(
             df_results=df, metrics_path=metrics_outputs_path, write_aggregate=None
         )
-        metrics_writer.parse_metrics_to_file(
-            df_results=df, metrics_path=metrics_outputs_path, write_aggregate="all"
-        )
 
-    def _create_infographics(self):
+        if self.hazard.event_mode != "risk":
```
[the remark this answers is not in the document]
Comment (dladrichem) [reply flood_adapt/object_model/direct_impacts.py:378]: Same thing, I didn't need it and didn't think anyone else was using it
